Gems/PhysX/Code/Source/HingeJointComponent.h

```diff
@@ -24,9 +26,24 @@ namespace PhysX
         HingeJointComponent(
             const JointComponentConfiguration& configuration, 
             const JointGenericProperties& genericProperties,
-            const JointLimitProperties& limitProperties);
+            const JointLimitProperties& limitProperties,
+            const JointMotorProperties& motorProperties = JointMotorProperties());
         ~HingeJointComponent() = default;
 
+        // JointInterfaceRequestBus::Handler
```

Comment:
nit: The preferred style for overrides is `// JointInterfaceRequestBus overrides ...` etc (super minor but would be great to update if possible)

Reply:
Change to recommended block comments.